.github/workflows/ci_container_build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build container
+
+on:
+  pull_request:
+    branches:
+      - main
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
You can get rid of these since they won't be needed for this workflow.

Reply:
Removed, thanks!